server/views/pages/induction/affectAbilityToWork/index.njk

```diff
@@ -0,0 +1,122 @@
+{% extends "../../../partials/layout.njk" %}
+
+{% set pageId = "induction-affect-ability-to-work" %}
+{% set title = "Is there anything that " + prisonerSummary.firstName + " " + prisonerSummary.lastName + " feels may affect their ability to work after they're released?" %}
+{% set pageTitle = title %}
+
+{#
+Data supplied to this template:
+  * prisonerSummary - object with firstName and lastName
+  * backLinkUrl - url of the back link
+  * backLinkAriaText - the aria label for the back link
+  * form - form object containing the following fields:
+    * affectAbilityToWork - Array existing values for field
+    * affectAbilityToWorkOther? - value for when Other is selected
+  * errors? - validation errors
+#}
+
+{% block beforeContent %}
+  {{ super() }}
+  {{ govukBackLink({ text: "Back", href: backLinkUrl, attributes: { "aria-label" : backLinkAriaText } }) }}
+{% endblock %}
+
+{% block content %}
+
+  {% if errors.length > 0 %}
+    {{ govukErrorSummary({
+      titleText: 'There is a problem',
+      errorList: errors,
+      attributes: { 'data-qa-errors': true }
+    }) }}
+  {% endif %}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      <form class="form" method="post" novalidate="">
+        <div class="govuk-form-group">
+          <input type="hidden" name="_csrf" value="{{ csrfToken }}" />
+
+          {% set otherHtml %}
+            {{ govukTextarea({
+              id: "affectAbilityToWorkOther",
+              name: "affectAbilityToWorkOther",
+              rows: "2",
+              value: affectAbilityToWorkOther,
+              type: "text",
+              label: {
+                text: "Give details",
+                attributes: { "aria-live": "polite" }
```

Comment:
This might be my lack of understanding, but what's the purpose of the `aria-live` attribute here? Is there dynamic content?

Reply:
We're copying these templates verbatim (or as close as we can) from the CIAG UI codebase, as we are trying to port these screens as "invisibly" / light touch as we can.
The content, including accessibility stuff (aria labels) has all been signed off and is in production, so for the purpose of this we are not looking to change anything like that.

In answer to your question of what is it's purpose / what is is meant to do / how does it work .... I have no idea 🤣  It's accessibility screen reader stuff! 🤣